src/couch_prometheus/src/couch_prometheus_server.erl

```diff
@@ -128,7 +130,14 @@ get_internal_replication_jobs_stat() ->
         internal_replication_jobs,
         gauge,
         "count of internal replication changes to process",
-        mem3_sync:get_backlog()
+        try mem3_sync:get_backlog() of
```

Comment:
If we return directly return the result and don't match on it, I think we can just to try `mem3_sync:get_backlog() catch ...` without the `of`?

Reply:
good idea; fixed in https://github.com/apache/couchdb/commit/7be4f02f269b183ab52950cea5c8f87a03b2eba7